index.js

```diff
@@ -38,14 +39,19 @@ const RNSearchablePicker = (({
     if (val.trim()) {
       // Filtered data
       const filtered = data.filter((item) => item.label.includes(val));
-      // Check if empty
       if (filtered.length) setFilteredData(filtered);
     } else {
       // Complete data withot filter 
       setFilteredData(data);
     }
   };
 
+  const onPress = () => {
+    // Open up list of items with filtered data empty
+    setListVisibility(!listVisibility); 
+    onChange("");
```

Comment:
Just wondering, is it necessary to call onChange with empty string?

Reply:
Should'nt but on my usage and tests here I had problems with list opened empty.